File: ASSIGNMENTS/Core java/Day02Assignment/src/com/sonata/TShirt.java
package com.sonata;

public class TShirt{
	
String color;
String material;
String design;
 String size;

 TShirt(){}
 TShirt(String color, String material, String design){
	this.color = color;
	this.material = material;
	this.design = design;
	}
 
	public void display() {
	System.out.println("color:" + color);
	System.out.println("material:" + material);
	System.out.println("design:" + design);
	}
	 

public static void main(String[] args) {
	
	TShirt tee  = new TShirt();  
	tee.size = "small";
	
	 
	TShirt teel  = new TShirt();  
	teel.size = "large";
	
	TShirt teex  = new TShirt();  
	teex.size = "xtralarge";
	
	TShirt t = new TShirt("red", "cotton", "solid");
	
	 t.display();
	System.out.println(tee.size);
	System.out.println(teel.size);
	System.out.println(teex.size);
	//System.out.println(tshirt.color);
	
	

}
}

File: ASSIGNMENTS/TypecriptAssignments/TShirt.js
var Tshirt = /** @class */ (function () {
    function Tshirt(c, m, d, s) {
        this.color = c;
        this.material = m;
        this.design = d;
        this.size = s;
    }
    Tshirt.prototype.display = function () {
        console.log(this.color + " " + this.material + " " + this.design + " " + this.size);
    };
    return Tshirt;
}());
var t1 = new Tshirt('pink', 'silk', 'solid', 'xxl');
var t2 = new Tshirt('black', 'cotton', 'printed', 'xxl');
var t3 = new Tshirt('blue', 'leather', 'stripes', 'm');
t1.display();
t2.display();
t3.display();
